contents/handbook/engineering/bug-prioritization.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Bug Prioritization
+sidebar: Handbook
+showTitle: true
+---
+
+When bugs are reported it's critical to properly gauge the extent and impact to be able to prioritize and respond accordingly. These are the priorities we use across the entire engineering org, along with the relevant labels to quickly identify them in GitHub.
+
+> Please always remember to tag your issues with the relevant priority.
+
+<span class="table-borders">
```

Comment:
I don't mind at all - but just letting you know for future reference that you don't need this class anymore